SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,73 @@
+Supported Operations
+####################
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Addition                                | - Falcon     | - 3                   |
+    |                  |                                         | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Subtraction                             | - Falcon     | - 2+                  |
+    |                  |                                         | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Multiplication                          | - Falcon     |                     3 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Mult, matmul, conv2d, conv2d _transpose | - Falcon     | - 3                   |
```

Comment:
I think there is repetition here multiplication is specified in above two columns